Makefile

```
@@ -137,6 +137,39 @@ deploy-postgres:
 	-kubectl wait --for=condition=ready pod -l app=pg-client -n postgres --timeout=120s
 	kubectl get pods -n postgres
 
+.PHONY: build-postgres-vuln
+build-postgres-vuln:
+	@echo "=== Building postgres-vuln image ==="
+	@DOCKER_HOST=$${DOCKER_HOST:-unix:///var/run/docker.sock}; export DOCKER_HOST; \
+	docker build -t postgres-vuln:latest postgres-vuln/
+
+.PHONY: deploy-postgres-vuln
+deploy-postgres-vuln: build-postgres-vuln
```

checkmake (0.3.2)

[warning] Target body for "deploy-postgres-vuln" exceeds allowed length of 5 lines (24).

(maxbodylength)